Assets/Scripts/Runtime/Scriptable Objects/Hand.cs

```diff
@@ -29,7 +29,17 @@ public void Initialize()
         public void SelectBlueprint(int chosenSegment)
         {
             _selection.Prefab = Option<Segment>.Some(SegmentsOptions[chosenSegment]);
-            _selection.Price = SegmentsOptions[chosenSegment].StaticSegmentData.ConnectionPoints.OpenConnectionPoints();
+
```

Comment:
Can you change it so static segment data has a field for price that we can use instead? We will soon have segments that are both blood and steam.